utils/epinow2/functions.py

```diff
@@ -207,6 +217,7 @@ def generate_task_configs(
                     "blob_storage_container": data_container,
                     "report_date": [report_date.isoformat()],
                     "reference_date": [x.isoformat() for x in reference_dates],
+                    "production_date": [production_date.isoformat()],
```

Comment:
Using the same array format as the other dates.

Reply:
So the idea with putting the dates in a list is that, at some point, we could pass in multiple dates, and it will spin up tasks for each production date?